lib/rss.js

```diff
@@ -23,6 +23,7 @@ const generateRssItem = (item) => {
   if (item.isJob) {
     title = item.title + ' \\ ' + item.company + ' \\ ' + `${item.location || ''}${item.location && item.remote ? ' or ' : ''}${item.remote ? 'Remote' : ''}`
   }
+  const category = item.subName ? `<category domain="${SITE_URL}/~${item.subName}">${item.subName}</category>` : '<category></category>'
```

Comment:
Even though this should never be relevant since only comments have no sub, I think it's better to always include tags and not conditionally include them.